src/Domain/Regulation/Enum/MeasureTypeEnum.php

```diff
@@ -6,7 +6,19 @@
 
 enum MeasureTypeEnum: string
 {
-    public const NO_ENTRY = 'noEntry';
-    public const ALTERNATE_ROAD = 'alternateRoad';
-    public const ONE_WAY_TRAFFIC = 'oneWayTraffic';
+    case NO_ENTRY = 'noEntry';
+    case ALTERNATE_ROAD = 'alternateRoad';
+    case ONE_WAY_TRAFFIC = 'oneWayTraffic';
+
+    public static function getFormChoices(): array
+    {
+        $values = array_column(self::cases(), 'value');
+        $choices = [];
+
+        foreach ($values as $value) {
+            $choices[sprintf('measures.types.%s', $value)] = $value;
```

Comment:
Finalement j'ai remis le scoping, par ex `regulation.measure.type.help`, on peut l'utiliser ici aussi

```suggestion
            $choices[sprintf('regulation.measure.type.%s', $value)] = $value;
```